dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/gdv/GdvFramework.kt

```diff
@@ -20,21 +20,42 @@ import org.dataland.frameworktoolbox.specific.uploadconfig.functional.FrameworkU
 import org.dataland.frameworktoolbox.specific.viewconfig.elements.getTypescriptFieldAccessor
 import org.dataland.frameworktoolbox.specific.viewconfig.functional.FrameworkDisplayValueLambda
 import org.dataland.frameworktoolbox.template.components.ComponentGenerationUtils
+import org.dataland.frameworktoolbox.utils.diagnostic.DiagnosticManager
 import org.springframework.stereotype.Component
 import java.io.File
 
 /**
  * Definition of the Heimathafen framework
  */
 @Component
-class GdvFramework : InDevelopmentPavedRoadFramework(
+class GdvFramework : PavedRoadFramework(
     identifier = "gdv",
     label = "GDV/VÖB",
     explanation = "Das GDV/VÖB Framework ist ein ESG-Fragebogen des Gesamtverbands der Versicherer und des " +
         "Bundesverbands Öffentlicher Banken",
     File("./dataland-framework-toolbox/inputs/gdv/dataDictionary-GDV-VOEB-GDV-VÖB ESG questionnaire.csv"),
 ) {
 
+    override fun configureDiagnostics(diagnosticManager: DiagnosticManager) {
```

Comment:
Are these errors still logged but not leading to failure? For me: To be tested

Reply:
I'd expect them not to be logged anymore.

The background of this is, that these are fields with some "Custom ***" entry in the "Component" column.
We do not want the devtools to parse these fields. We want to manually create them in the customization hook.
In order to make the devtools not say "hey, wtf is Custom*** ?  I don't know such a component" we need these ignores here.

Reply:
TODO for me. Check this function further.